### DISCUSSION TITLE
morphmap for relationship

### DISCUSSION THREAD

**Discussion by fullyloadedtech** [awcodes/filament-curator#208] [Ideas]
When using something like `CuratorPicker::make('photos')->multiple()->relationship('photos', 'id')` I would prefer if there was a way to use a morphmap/"morph many" instead of needing to create a pivot table for every relation. As in potentially use a single table for multiple relationships, rather than multiple pivot tables.

**Comment by awcodes** [comment]
Can you provide an example? I'm thinking that would be a breaking change which I'm not prepared to do with v3 having to be a major version change anyway.
